packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -798,22 +796,42 @@ function FirstTermDiscountCallout( { product }: { product: ResponseCartProduct }
 		// Do not display discount reason if a coupon is applied.
 		isCouponApplied( product )
 	) {
-		return null;
+		return <></>;
 	}
-
+	let oneTimeExplanation: ReactChild = '';
 	if ( isMonthlyProduct( product ) ) {
-		return <DiscountCallout>{ translate( 'Discount for first month' ) }</DiscountCallout>;
+		oneTimeExplanation = translate( 'Discount: %(discount)s Applied in first month only', {
```

Comment:
Should we use `hasTranslation` on these so that they can be merged before the new strings are translated?

Reply:
I was actually planning to string freeze this change after a confidence check from a reviewer 👍

Reply:
Sounds good!